packages/grafana-ui/src/components/Table/TableInputCSV.story.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import { storiesOf } from '@storybook/react';
+import TableInputCSV, { ParseResults } from './TableInputCSV';
+
+const TableInputStories = storiesOf('UI/Table/Input', module);
+
+TableInputStories.add('default', () => {
+  return (
+    <div>
+      <TableInputCSV
+        width={'90%'}
+        height={'90vh'}
+        onTableParsed={(results: ParseResults) => {
+          console.log('GOT', results);
```

Comment:
I suggest using storybook's action addon for such cases. You don't even have to open console then, as everything is logged in the Actions pane below story (i.e. https://github.com/grafana/grafana/blob/master/packages/grafana-ui/src/components/ColorPicker/ColorPicker.story.tsx#L30)

Reply:
nice.  That is much better:
![image](https://user-images.githubusercontent.com/705951/54074333-1771f900-4246-11e9-8059-51b1e1154e84.png)